runtimevar/awsparamstore/paramstore_test.go

```diff
@@ -173,13 +173,26 @@ func TestOpenVariable(t *testing.T) {
 		{"awsparamstore://myvar?decoder=notadecoder", true},
 		// Invalid parameter.
 		{"awsparamstore://myvar?param=value", true},
+		// Decrypt+bytes decoder.
+		{"awsparamstore://mysecretvar?decoder=decrypt/bytes", false},
 	}
 
 	ctx := context.Background()
+	cleanup := setupTestSecrets(ctx)
+	defer cleanup()
+
 	for _, test := range tests {
 		_, err := runtimevar.OpenVariable(ctx, test.URL)
 		if (err != nil) != test.WantErr {
 			t.Errorf("%s: got error %v, want error %v", test.URL, err, test.WantErr)
 		}
 	}
 }
+
+func setupTestSecrets(ctx context.Context) func() {
```

Comment:
Instead of doing this full test in `awsparamstore`, how about doing it in `constantvar` or `filevar` (with `localsecrets`) so that you can verify the output as well?

Reply:
Oh, I see you did do it in filevar too; I would just remove from here then.